src/index.html

```diff
@@ -7,12 +7,63 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
```

Comment:
The second preconnect link to 'https://fonts.googleapis.com' is redundant as it is already declared above. You can remove this duplicate link to optimize the loading performance.